src/features/email/component/email-sidebar/email-sidebar.tsx

```diff
@@ -56,28 +71,46 @@ export function EmailSidebar() {
   ];
 
   return (
-    <div className="flex min-w-[280px] flex-col ">
-      <div className=" p-4">
-        <h2 className="text-2xl font-bold tracking-tight">Mail</h2>
-      </div>
-      <div className="py-4 px-2">
-        <Button variant="default" className="flex items-center w-full">
-          <SquarePen size={20} />
-          Compose
-        </Button>
-      </div>
-      <div className="flex-1 px-2">
-        {navItems.map((item, index) => (
-          <NavItem key={index} {...item} />
-        ))}
+    <>
+      <div className="flex min-w-[280px] flex-col ">
+        <div className=" p-4">
+          <h2 className="text-2xl font-bold tracking-tight">Mail</h2>
+        </div>
+        <div className="py-4 px-2">
+          <Button
+            variant="default"
```

Comment:
No need to pass the props variant="default" as in Button component default variant itself is "default"